src/test/java/org/testeditor/fixture/web/drivermanager/InternetExplorerTest.java

```diff
@@ -0,0 +1,138 @@
+package org.testeditor.fixture.web.drivermanager;
+
+import io.github.bonigarcia.wdm.BrowserManager;
+import io.github.bonigarcia.wdm.InternetExplorerDriverManager;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Properties;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class InternetExplorerTest {
+
+    private WebDriver driver;
+
+    private static String proxyHost;
+    private static String proxyUser;
+    private static String proxyPassword;
+    private static String testSiteUrl;
+    private static String searchfieldID;
+    private static String searchString;
+    private static String firstSiteTitle;
+    private static String xPathResult;
+    private static String resultSiteTitle;
+
+    @BeforeClass
+    public static void setupClass() throws IOException {
+        assumeWindowsPresent();
+        BrowserManager browserManager = InternetExplorerDriverManager.getInstance();
+        browserManager.proxy(proxyHost);
+        browserManager.proxyUser(proxyUser);
+        browserManager.proxyPass(proxyPassword);
+        browserManager.setup();
+    }
+
+    @Before
+    public void setupTest() throws IOException {
+        DesiredCapabilities capabilities = DesiredCapabilities.internetExplorer();
+        capabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
```

Comment:
is this special for some customers? should this be passed via env variable to be configured?